services/121-service/src/payments/dto/transaction-response.dto.ts

```diff
@@ -0,0 +1,41 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+import { TransactionStatusEnum } from '@121-service/src/payments/transactions/enums/transaction-status.enum';
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+
+export class TransactionResponseDto {
+  @ApiProperty({ example: 1, type: 'number' })
+  public id: number;
+  @ApiProperty({
+    example: '2023-09-28T08:00:10.363Z',
+    type: 'string',
+    format: 'date-time',
+  })
+  public created: Date;
+  @ApiProperty({
+    example: '2023-09-28T08:00:10.363Z',
+    type: 'string',
+    format: 'date-time',
+  })
+  public updated: Date;
```

Comment:
Flat list of key-value pairs or nested object for Registration? => Architecture chit-chat to have a guideline.

Reply:
I added it to the parking lot.